instance-conf/httpd-2.4/bin/fixrootpath.ps1

```diff
@@ -18,26 +18,36 @@
 
 # fixrootpath.ps1
 #
+# This script searches and replaces the installation paths specified by:
+#
+#   --srcdir=path or string to be replaced 
+#   --dstdir=/path/to/destination/installation
+#
+# This package of Apache HTTP Server include two patterns to replace:
```

Comment:
"This package of Apache HTTP Server include"
should be
"This package of Apache HTTP Server includes"